CHANGELOG.md

```diff
@@ -1,6 +1,102 @@
 CHANGELOG
 =========
 
+v2.0.0
+-------------
+
+### Summary
+
+* New tilequeue commands have been added to help deal with common issues:
+* * `status` returns information about whether the tile is present in storage, the TOI and the in-flight list. This can help diagnosing issues with bad or stale tiles being returned.
+* * `stuck-tiles` lists tiles which are present in storage but not in the TOI. These might be causing problems, since they will not be updated. `delete-stuck-tiles` takes the list output by `stuck-tiles` and deletes them.
+* Database queries are now per-table rather than per-layer. This can help efficiency slightly, as the database may be able to optimise the number of blocks read from disk into a single pass. However, the main reason was to prepare for RAWR tiles, which are collected into per-table files.
```

Comment:
Link to RAWR tiles repo, I think `which are collected into per-tale files` summarizes RAWR tiles well so no more explanation is needed besides adding the link.